index.d.ts

```diff
@@ -49,7 +58,7 @@ declare class QuickLRU<KeyType, ValueType> implements Iterable<[KeyType, ValueTy
 
 	@returns The list instance.
 	*/
-	set(key: KeyType, value: ValueType): this;
+	set(key: KeyType, value: ValueType, expiry?: number): this;
```

Comment:
This should be `maxAge` for consistency and needs to be documented.

Reply:
Done and documented let me know if its' fine for you